### PR TITLE
[DOCFIX] Make passive.cache more concrete and easier to understand.

### PR DESCRIPTION
**Pull request by maobaolong**
We want the user easier to know if `alluxio.user.file.passive.cache.enabled` option turn on, client which have a local worker can cache the file's blocks when reading from remote workers, this can reduce the remote worker reading, because of the local worker have owned this file's block. But, it have to increase the usage of the Alluxio tiered store.

### CODE REVIEW

File: docs/_data/table/cn/user-configuration.yml [hunk 1]
```diff
@@ -36,6 +36,8 @@ alluxio.user.file.write.avoid.eviction.policy.reserved.size.bytes:
   当用户选择LocalFirstAvoidEvictionPolicy作为写文件数据块的定位机制时,用户需要配置worker预留一些数据量来保证数据的存储，默认是0MB。
 alluxio.user.file.write.tier.default:
   数据块写入的默认存储层。可选值为整型数值。非负值代表从高层到底层的存储层（0代表第一层存储层，１代表第二层存储层，以此类推）。如果给定值大于存储层数量,这个数字代表最底层的存储层。负值代表从底层到高层的存储层（-1代表最底层存储层，-2代表次底层存储层，以此类推）如果给定值的绝对值大于存储层数量，这个数字代表最高层存储层。
+alluxio.user.file.passive.cache.enabled:
```
Comment (yupeng9) [inline docs/_data/table/cn/user-configuration.yml]: move this up to be after `alluxio.user.file.metadata.load.type` so the keys are alphabetically sorted.

File: docs/_data/table/en/user-configuration.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ alluxio.user.file.master.client.threads:
 alluxio.user.file.metadata.load.type:
   'The behavior of loading metadata from UFS. When information about a path is requested and the path does not exist in Alluxio, metadata can be loaded from the UFS. Valid options are `Always`, `Never`, and `Once`. `Always` will always access UFS to see if the path exists in the UFS. `Never` will never consult the UFS. `Once` will access the UFS the "first" time (according to a cache), but not after that.'
 alluxio.user.file.passive.cache.enabled:
-  'Whether to cache files when reading.'
+  'Whether to cache files to local Alluxio workers when the files are read from remote workers (not UFS).'
```
Comment (yupeng9) [inline docs/_data/table/en/user-configuration.yml]: I think this option also applies to read from UFS. When it's on, reading from UFS will also cache to local worker

Comment (maobaolong) [reply docs/_data/table/en/user-configuration.yml]: Yeah, so i'd better to make more concrete.

Comment (yupeng9) [reply docs/_data/table/en/user-configuration.yml]: did you forget to push the commit for this?

Comment (yupeng9) [reply docs/_data/table/en/user-configuration.yml]: @maobaolong  ping?

Comment (maobaolong) [reply docs/_data/table/en/user-configuration.yml]: @yupeng9 I've update this PR, PTAL.

File: docs/_data/table/cn/user-configuration.yml (hunk 1 above)
Comment (maobaolong) [reply docs/_data/table/cn/user-configuration.yml]: Done. i've sort this file.

**Pull request merged**